examples/general/simple-web/README.md

```diff
@@ -2,3 +2,20 @@ An absolute bare-bones web app.
 
 Created from templates made available by Stagehand under a BSD-style
 [license](https://github.com/dart-lang/stagehand/blob/master/LICENSE).
+
+Run this example first to make sure all your tooling is working.
+
+## Detailed instructions (as of Nov2019) for Android Studio
+
+0. Clone the github repo for the `pwa` package.
+1. Install the Dart SDK (https://dart.dev/get-dart) (if you haven't already). This is different from the Flutter SDK.
+2. Run `brew info dart` in terminal and confirm no errors. Note the path to the Dart SDK. On mac OSX, this was `/usr/local/opt/dart/libexec`; Or, note where your Dart SDK is installed if you used another method.
+3. In Android Studio, open up the `simple-web` folder as a project. (Not just navigating to it from the main package).
```

Comment:
I don't think you'll need Android Studio for this at all. (Also section title can be updated too)

Reply:
I don't feel strongly about this. As a junior developer and someone new to dart/flutter, I find step-by-step examples to be really helpful.

I clarified this with a sentence:
`If you already know what `webdev` and have Dart SDK, you may just be able to directly run the example based on your own knowledge of the development environment. If you are confused, here is one possible route to making this example work.`

I'm fine with you making any changes you want (and grateful you wrote this whole package!). But I'm a fine of the step-by-step that is helpful for more junior people.